src/renderer/components/App/AppSidemenu/AppSidemenu.vue

```diff
@@ -54,22 +54,21 @@
           />
 
           <!-- Plugins -->
-          <AppSidemenuPlugins
-            v-if="hasPluginMenuItems && isPluginsVisible"
-            :outside-click="true"
-            :is-horizontal="isHorizontal"
-            @close="closeShowPlugins"
-          />
-
           <MenuNavigationItem
-            v-if="hasPluginMenuItems"
```

Comment:
Now this is not displaying the menu items defined in plugins

Reply:
I thought that but presumed it's because i had an outdated plugin so was going to use an example plugin today. Thanks for double checking!